pallets/avn/src/lib.rs

```diff
@@ -146,6 +200,18 @@ impl<T: Config> Pallet<T> {
         Ok(maybe_validator.expect("Already checked"))
     }
 
+    pub fn calculate_eth_tx_expiry() -> U256 {
+        let now = T::TimeProvider::now();
+        let now_as_secs = now.as_secs();
+
+        let timeout_duration_hours = Self::get_eth_tx_timeout_duration();
+        let timeout_duration_seconds = timeout_duration_hours * 3600;
```

Comment:
Could we make the storage value seconds instead? This would afford more fine-grained control

Reply:
sure will remove the calculation and will pass the value of the storage item directly